src/main/java/org/alliancegenome/curation_api/services/AllelePhenotypeAnnotationService.java

```diff
@@ -0,0 +1,95 @@
+package org.alliancegenome.curation_api.services;
+
+import java.util.List;
+
+import org.alliancegenome.curation_api.dao.AllelePhenotypeAnnotationDAO;
+import org.alliancegenome.curation_api.dao.ConditionRelationDAO;
+import org.alliancegenome.curation_api.enums.BackendBulkDataProvider;
+import org.alliancegenome.curation_api.exceptions.ObjectUpdateException;
+import org.alliancegenome.curation_api.model.entities.AllelePhenotypeAnnotation;
+import org.alliancegenome.curation_api.model.entities.Allele;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.PhenotypeFmsDTO;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.services.base.BaseEntityCrudService;
+import org.alliancegenome.curation_api.services.validation.dto.fms.AllelePhenotypeAnnotationFmsDTOValidator;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+
+@RequestScoped
+public class AllelePhenotypeAnnotationService extends BaseEntityCrudService<AllelePhenotypeAnnotation, AllelePhenotypeAnnotationDAO> {
+    
+    @Inject
+	AllelePhenotypeAnnotationDAO allelePhenotypeAnnotationDAO;
+	@Inject
+	ConditionRelationDAO conditionRelationDAO;
+	@Inject
+	PhenotypeAnnotationService phenotypeAnnotationService;
+	@Inject
+	AllelePhenotypeAnnotationFmsDTOValidator allelePhenotypeAnnotationFmsDtoValidator;
+
+	@Override
+	@PostConstruct
+	protected void init() {
+		setSQLDao(allelePhenotypeAnnotationDAO);
+	}
+
+	public ObjectResponse<AllelePhenotypeAnnotation> get(String identifier) {
```

Comment:
@markquintontulloch doesn't this method exists in the submitted object class?

Reply:
it would be good to have this logic in one place

Reply:
SubmittedObject classes don't have uniqueId.  We should be able to refactor this though as there is duplication in all the DiseaseAnnotation and PhenotypeAnnotation classes.

Reply:
Marking this as resolved, as this will be handled during refactoring

Reply:
why not refactor now? as refactoring is never a priority?

Reply:
@oblodgett I’ll do the refactor in a separate PR